pkg/minishift/registration/redhat.go

```diff
@@ -87,11 +92,15 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 				param.Username,
 				minishiftStrings.EscapeSingleQuote(param.Password))
 
-			// start timed SSH command to register
-			spinnerView.Start()
+			if spinnerView != nil {
```

Comment:
and if done so, we do not check for `if spinnerView != nil`, but only need to concern about `if ShowProgressBars { ... }`